README.md

```diff
@@ -41,6 +41,10 @@ TOMtoolkit module for reporting transients to the TNS
                 'c': 'Clear',
                 ...
             },  # Optional mapping from your reduced datum filter values to TNS filter options.
+            'instrument_mapping': {
```

Comment:
Can we be more specific about what these mappings do?
Maybe it would make sense to have a small section that describes how to use the filter/instrument mapping and that this will try to automatically translate your values in Reduced Datums into filled fields.
We should be explicit about which fields in the JSON need to exist for this to work.

Reply:
The comment below I thought was enough specificity (reduced datum instrument name to TNS instrument name), but if you have other ideas on what to say I'm happy to change it. I guess I can add a section to the readme about what the default tom_base implementation tries to pull out field-wise, but really its up to each individual tom user to implement their own data converter to match their fields.